webapp/src/js/services/format-data-record.js

```diff
@@ -180,13 +180,36 @@ angular.module('inboxServices').factory('FormatDataRecord',
               });
     };
 
-    var formatDate = function(settings, date) {
+    var capitalizeWord = function(str) {
+      return str.charAt(0).toUpperCase() + str.slice(1);
+    };
```

Comment:
Don't capitalize a translated value - let the configurer decide if it should be capitalized.